### PR TITLE
Updated missing types in help and docs

### PR DESCRIPTION
**Pull request by sabre1041**
Updated valid image types based on the current supported options

### CODE REVIEW

**Comment by cgwalters** [comment]
Ah we were doing the same thing at the same time, see also https://github.com/osbuild/bootc-image-builder/pull/269

File: README.md [hunk 1]
```diff
@@ -158,10 +158,10 @@ The following image types are currently available via the `--type` argument:
 
 | Image type            | Target environment                                                                    |
 |-----------------------|---------------------------------------------------------------------------------------|
-| `ami`                 | [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) |
+| `ami`, `raw`          | [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) |
```
Comment (cgwalters) [inline README.md:161]: `.raw` is not specific to any cloud.

File: README.md [hunk 2]
```diff
@@ -158,10 +158,10 @@ The following image types are currently available via the `--type` argument:
...
 | `qcow2` **(default)** | [QEMU](https://www.qemu.org/)                                                         |
 | `vmdk`                | [VMDK](https://en.wikipedia.org/wiki/VMDK) usable in vSphere, among others            |
-| `anaconda-iso`        | An unattended Anaconda installer that installs to the first disk found.               |
+| `iso`, `anaconda-iso` | An unattended Anaconda installer that installs to the first disk found.               |
```
Comment (cgwalters) [inline README.md:164]: Conceptually `iso` is a now-deprecated alias for `anaconda-iso`; I think we should continue to omit it from the help output, or even just remove it entirely now.

File: bib/cmd/bootc-image-builder/main.go [hunk 3]
```diff
@@ -407,7 +407,7 @@ func run() error {
 	manifestCmd.Flags().String("config", "", "build config file")
 	manifestCmd.Flags().String("rpmmd", "/rpmmd", "rpm metadata cache directory")
 	manifestCmd.Flags().String("target-arch", "", "build for the given target architecture (experimental)")
-	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, "image types to build [qcow2, ami, iso, raw]")
+	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, "image types to build [qcow2, ami, iso, raw, vmdk, anaconda-iso]")
```
Comment (cgwalters) [inline bib/cmd/bootc-image-builder/main.go:410]: This one makes total sense, but I think https://github.com/osbuild/bootc-image-builder/pull/269 will get us to a slightly more sustainable place.  That said now looking at it I realize I should also update the above for the Go error place too.

Comment (sabre1041) [reply bib/cmd/bootc-image-builder/main.go]: @cgwalters i can incorporate your implementation if that helps

Comment (cgwalters) [reply bib/cmd/bootc-image-builder/main.go]: I'm not sure we want the changes to README.md; do you agree with my comments?

Comment (cgwalters) [reply bib/cmd/bootc-image-builder/main.go]: Sure, feel free to pull in my PR changes into this!

Comment (sabre1041) [reply bib/cmd/bootc-image-builder/main.go]: @cgwalters done

**Review by cgwalters — APPROVED** [review]

**Pull request merged**